### DISCUSSION TITLE
Virtual IP incase of reauthentication

### DISCUSSION THREAD

**Discussion by nknkgithub** [strongswan/strongswan#2252] [Q&A]
We want to use reauthentication feature in strongswan 5.9.14.
We are setting make_before_break = yes in strongswan.conf and reauth_time > 0 in swanctl.conf in client side.
In server side(security gateway) no change is done.

What is the expectation of virtual IP from server?
Does server suppose to allocate the same virtual IP on every reauth or different IP?

**Comment by nknkgithub** [comment]
Help required!!

**Comment by tobiasbrunner** [comment]
> What is the expectation of virtual IP from server?

What do you mean?

> Does server suppose to allocate the same virtual IP on every reauth or different IP?

If you are referring to the in-memory pools (e.g. via `pools` section in swanctl.conf) then the same virtual IP sould get allocated to the same client.
